lib/govuk_content_models/presentation_toggles.rb

```diff
@@ -0,0 +1,39 @@
+module PresentationToggles
+  extend ActiveSupport::Concern
+
+  included do
+    field :presentation_toggles, type: Hash, default: default_presentation_toggles
+    validates_presence_of :organ_donor_registration_promotion_url, if: :display_organ_donor_registration_promotion?
+  end
+
+  def display_organ_donor_registration_promotion=(value)
+    value = value.is_a?(Boolean) ? value : value != '0' # if assigned using a checkbox
+    organ_donor_registration_promotion_key['display'] = value
+  end
+
+  def display_organ_donor_registration_promotion
+    organ_donor_registration_promotion_key['display']
+  end
+  alias_method :display_organ_donor_registration_promotion?, :display_organ_donor_registration_promotion
+
+  def organ_donor_registration_promotion_url=(value)
+    organ_donor_registration_promotion_key['url'] = value
+  end
+
+  def organ_donor_registration_promotion_url
+    return "" unless display_organ_donor_registration_promotion?
+    organ_donor_registration_promotion_key['url']
+  end
+
+  def organ_donor_registration_promotion_key
+    presentation_toggles['organ_donor_registration_promotion']
+  end
+
+  module ClassMethods
+    def default_presentation_toggles
+      {
+        'organ_donor_registration_promotion' => { 'url' => '', 'display' => false }
+      }
+    end
+  end
+end
```

Comment:
Hi Jamie, i've done something like this before when i worked on [the beta label story](https://github.com/alphagov/govuk_content_models/pull/210/files). if i would have done something like presentation toggles at that time, it would have saved us some effort now. such changes require content-api and frontend also to be updated to handle new fields. it'll be good to ship these fields as `presentation_toggles` to reduce changes in those apps. right now, content-api doesn't have to worry about what'll is involved in getting organ donor registration to be displayed; as it should be.

it is a non-trivial task to migrate `in_beta` flag into `presentation_toggles`; which serves as a strong reason to have these toggles even if we have only organ donor registration to deal with currently.

> it would take a fair bit of code change to support each additional field
if we add more toggles, we'd use some meta-programming to generate these methods. i don't see this module growing too large.

let me know whether i'm making sense.